src/address_format.rs

```diff
@@ -105,10 +105,13 @@ impl<'a> TryFrom<&'a str> for Ss58AddressFormatRegistry {
 	type Error = ParseError;
 
 	fn try_from(x: &'a str) -> Result<Ss58AddressFormatRegistry, Self::Error> {
+		let x = x.to_ascii_lowercase();
```

Comment:
Why not `to_lowercase`?

Reply:
I think there won't be a weird name.

But okay, let's make it `to_lowercase`.